charts/unauthenticated-ingress/README.md

```diff
@@ -0,0 +1,127 @@
+standard-application
```

Comment:
remove all references to `standard-application` and replace with `unauthenticated_ingress`